Commands/Base/EnablePowerShellTelemetry.cs

```diff
@@ -12,6 +12,10 @@ namespace SharePointPnP.PowerShell.Commands.Base
        Code = "PS:> Enable-PnPPowerShellTelemetry",
        Remarks = "Will prompt you to confirm to enable telemetry tracking.",
        SortOrder = 1)]
+    [CmdletExample(
+       Code = "PS:> Enable-PnPPowerShellTelemetry",
```

Comment:
Did you miss a -Force in the example here?

Reply:
Not sure what you mean. Are you saying it doesn't work without force, that it should work already with -Force without my code change or something else? :)

Reply:
The description says it will not prompt you, but the example doesn't use the -Force parameter